doc/api_conf.rst

```diff
@@ -88,3 +88,10 @@ Configurable settings of the :class:`dnf.Base` object are stored into a :class:`
     Update or create config file. Where `filename` represents name of config file (.conf or .repo); `section_id`
     represents id of modified section (e.g. main, fedora, updates); `substitutions` represents an instance of
     base.conf.substitutions; `modify` represents dict of modified options.
+
+
+.. class:: dnf.conf.substitutions.Substitutions
+
+  .. method:: update_from_etc(installroot, varsdir=("/etc/yum/vars/", "/etc/dnf/vars/"))
+
+    Read user-defined variables values from variable directories. See :ref:`variable files <varfiles-label>` in Configuration reference.
```

Comment:
LGTM, just the `variable files` documentation linked here doesn't mention `/etc/yum/vars/` and that's what's used in the bug, so maybe verify that and add it to the documentation too? (perhaps with a deprecation notice...)

Reply:
I added a note about `/etc/yum/vars` to variable files section.